option optimize_for = LITE_RUNTIME;

message FashionDressRequest
{
 required uint32 pos = 1;
}
